### DISCUSSION TITLE
Now that Mocaccino is beyond Beta...

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#93] [General] [answered]
Hello to all Mocaccino gurus!    
Is it now safe to try to dual-boot with Mocaccino? 
I asked this question back in or around June and was told it was NOT safe being that Mocaccino was still beta.  
Assume that the system open for this experimentation currently has:
```
Device     Boot  Size   Type
/dev/sda1  *     100M   HPFS/NTFS/exFAT
/dev/sda2        390.5G HPFS/NTFS/exFAT
/dev/sda3        540.9G Linux
```
I do not want to destroy the data resident in /dev/sda2 so it must remain accessible and functional within that Windows OS. 
Thank you very much in advance.

**Comment by joostruis** [comment, marked as answer]
I dual boot my PC with Windows. Grub adds an entry for Windows on my PC.

**Comment by msdobrescu** [comment]
Here works too, also having Sabayon too (for reference to my tools settings...). Anyway, looking over the Internet for `Warning: os-prober will not be executed to detect other bootable partitions.` shows that there's a bug in grub causing many distros to disable detecting other OSes installations. It's a grub bug in certain conditions that affects any distro that uses it. You either keep `os-prober` disabled and add yourself the entries manually, either let it detect the other bootable OSes and if it works for you, let it enabled. Beware, if the bug hits you, _may_ break safe boot, apparently, so have a backup first.

**Comment by VolvoBirkenstock** [comment]
Thank you @joostruis and @msdobrescu.  

I will attempt to replace Sabayon on that dual-bootable system at my earliest opportunity.  It will take a while since I'm buried with work and domestic tasks. :-)  I will definitely get back to you with the results.  I also need to learn more about luet and how to find packages that replace the functionality currently loaded from Sabayon.  

Again, thank you VERY much.
